<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 18pt &quot;Dubai Medium&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>UNIT OF ACTIVITY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;font: 12pt &quot;Dubai&quot;</string>
    </property>
    <property name="text">
     <string>Harga Awal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Dubai&quot;;
color:black;
</string>
    </property>
    <property name="text">
     <string>Nilai Residu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Dubai&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>Pemakaian</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Harga">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:black;
color:black;</string>
    </property>
    <property name="placeholderText">
     <string>Rp. </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Nilai">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:black;
color:black;</string>
    </property>
    <property name="placeholderText">
     <string>Rp. </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Pemakaian">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:black;
color:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Dubai&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>Kapasitas Maksimal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Kapasitas">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:black;
color:black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Hitung">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Hitung</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>left-arrow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Dubai&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>Hasil</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Hasil">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:black;
color:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icon/left-arrow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
